app/services/witter.js

```diff
@@ -0,0 +1,10 @@
+import axios from 'axios';
+
+export async function weet() {
+  try {
+    const response = await axios.get('https://quote-garden.herokuapp.com/api/v3/quotes/random');
+    return response.data.data;
```

Comment:
So that you don't have to do so much nesting you could do destructuring to the API response and maybe so that the response is not `data.data` you could add an alias to it.